Base/home/anon/.config/bookmarks.json

```diff
@@ -1,38 +0,0 @@
-[
```

Comment:
we should move this to somewhere so we can add bookmarks to the current chromes, instead of deleting it.

Reply:
Could also be part of the default config template idea